osutil/mountentry_linux_test.go

```diff
@@ -57,6 +57,11 @@ func (s *entrySuite) TestString(c *C) {
 		Options: []string{"rw,noatime"},
 	}
 	c.Assert(ent3.String(), Equals, `/dev/sda5 /media/My\040Files ext4 rw,noatime 0 0`)
+	ent4 := osutil.MountEntry{
+		Dir:     "$HOME/.local/share",
+		Options: []string{"x-snapd.kind=ensure-dir", "x-snapd.must-exist-dir=$HOME"},
+	}
+	c.Assert(ent4.String(), Equals, "none $HOME/.local/share none x-snapd.kind=ensure-dir,x-snapd.must-exist-dir=$HOME 0 0")
 }
```

Comment:
Could you for comparison add a similar test for the symlink case, afaict there is no such preexisting test

Reply:
Done